pkg/pgmodel/ingestor/ingestor_interface.go

```diff
@@ -18,5 +19,6 @@ type DBInserter interface {
 	// Returns the number of metrics ingested and any error encountered before finishing.
 	Ingest(context.Context, *prompb.WriteRequest) (uint64, uint64, error)
 	IngestTraces(context.Context, ptrace.Traces) error
+	IngestJaegerSpan(ctx context.Context, span *model.Span) error
```

Comment:
Since `ingestor` works with already defined OTEL trace model I'd recommend to not add Jaeger specific function here. We can have `translator` code done before (eg in the API layer ).

Reply:
Sure, I can do that. Thats how I started, but had an afterthought of keeping jaeger specific ingest function might help incase if we want to do any optimisation in future. But I agree, it can't be designed as future proof and we can introduce an jaeger specific interface function when the need arises.

This is as simple as reverting https://github.com/timescale/promscale/pull/1543/commits/23d899c58ccfb98e0cbf90695713ae9374faa789. I will do it anyway :)